tests/test_derivative.py

```diff
@@ -33,6 +33,7 @@ def circuit(nqubits=1):
 def test_standard_parameter_shift(backend, nshots, atol, scale_factor, grads):
     # initializing the circuit
     c = circuit(nqubits=1)
+    np.random.seed(42)
```

Comment:
```suggestion
    backend.set_seed(42)
```

Maybe this is more general